sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/__init__.py

```diff
@@ -48,7 +50,9 @@
     'LinkedEntityMatch',
     'TextDocumentBatchStatistics',
     'SentenceSentiment',
-    'SentimentConfidenceScores'
+    'SentimentConfidenceScores',
+    'AspectSentiment',
+    'OpinionSentiment'
```

Comment:
Super nitpick - put a trailing comma here. It helps avoid problems when we add more elements to the array. 

If you are curious as to why that is generally a good idea, consider how python interprets this:

```python
[
    'hello'
    'world'
]
```